core-bundle/src/Doctrine/Schema/DcaSchemaProvider.php

```diff
@@ -316,7 +313,20 @@ private function parseIndexSql(Table $table, $keyName, $sql)
                 $flags[] = 'fulltext';
             }
 
-            $table->addIndex($columns, $matches[2], $flags);
+            $options = [];
+
+            if (array_sum($lengths)) {
```

Comment:
This is nice. 👍 Although I really wonder why null values do not trigger a warning with `declare(strict_types=1)`.

Reply:
We could also use

```suggestion
            if (array_filter($lengths)) {
```

instead. Should we?

Reply:
Yes.

Reply:
Done in edb3c933828520f7c7bac39f4d2e060fa184950f